client/partials/abtAppropriations.html

```diff
@@ -0,0 +1,3 @@
+<div class="abtAppropriations">
```

Comment:
What are your thoughts on class naming? Most css/html standards say to use all lowercase with words separated by hyphens, and some html linters will throw an error if it's not. I'm fine with it being this way, just wanted to know your thoughts.